conf/hiera.yaml

```diff
@@ -0,0 +1,10 @@
+---
+# Hiera 5 Global configuration file
+
+version: 5
+
+# defaults:
+#   data_hash: yaml_data
+# hierarchy:
+#  - name: Common
+#    data_hash: yaml_data
```

Comment:
A warning here. If the intention is that this should provide a completely empty config, then it must be equipped with an empty hierarchy. Without it, it will get the default hierarchy and, as described here, the default isn't empty.

Suggest adding:
```
hierarchy: []
```
to explicitly set the hierarchy to an empty array.